students/wachulec_maria/lesson_06_dicts_tuples_sets_args_kwargs/fantasy_game.py

```diff
@@ -0,0 +1,20 @@
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    for k, v in inventory.items():
+        print(v, k)
+        item_total += v
+    print("Total number of items: " + str(item_total))
+
+
+def add_to_inventory(inventory, added_items):
+    for key in added_items:
+        inventory.setdefault(key, 0)
```

Comment:
good. It's also worth to know about `defaultdict ` https://www.accelebrate.com/blog/using-defaultdict-python/
and
`.get()` method http://www.tutorialspoint.com/python/dictionary_get.htm